tests/clients/test_experiment.py

```diff
@@ -105,33 +106,52 @@
 METRIC_TIMESTAMP = datetime(2018, 3, 12, 16, 20, 22, 122000, tzinfo=UTC)
 METRIC_TIMESTAMP_STRING_JAVA = "2018-03-12T16:20:22.122Z"
 METRIC_TIMESTAMP_STRING_PYTHON = "2018-03-12T16:20:22.122000+00:00"
-METRIC = Metric(key=METRIC_KEY, value=123, timestamp=METRIC_TIMESTAMP)
+METRIC = Metric(key=METRIC_KEY, value=123, timestamp=METRIC_TIMESTAMP, step=0)
 METRIC_BODY = {
     "key": METRIC_KEY,
     "value": 123.0,
     "timestamp": METRIC_TIMESTAMP_STRING_PYTHON,
+    "step": 0,
+}
+
+METRIC_WITHOUT_STEP = Metric(
+    key=METRIC_KEY, value=123, timestamp=METRIC_TIMESTAMP, step=None
+)
+METRIC_WITHOUT_STEP_BODY = {
+    "key": METRIC_KEY,
+    "value": 123.0,
+    "timestamp": METRIC_TIMESTAMP_STRING_PYTHON,
+    "step": None,
 }
 
 METRIC_SECOND_TIMESTAMP = datetime(2018, 3, 12, 16, 20, 30, 122000, tzinfo=UTC)
 METRIC_SECOND_TIMESTAMP_STRING_PYTHON = "2018-03-12T16:20:30.122000+00:00"
 METRIC_SECOND_MEASURE = Metric(
-    key=METRIC_KEY, value=127, timestamp=METRIC_SECOND_TIMESTAMP
+    key=METRIC_KEY, value=127, timestamp=METRIC_SECOND_TIMESTAMP, step=0
```

Comment:
To make this case more different from `METRIC`:
```suggestion
    key=METRIC_KEY, value=127, timestamp=METRIC_SECOND_TIMESTAMP, step=1
```